src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/allocatecase/AllocateTheCaseMidEventHandler.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit.allocatecase;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.CASE_WORKER_NAME_LIST;
+
+import java.util.List;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.DynamicList;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.Value;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.Callback;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.callback.PreSubmitCallbackStage;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.PreSubmitCallbackHandler;
+import uk.gov.hmcts.reform.iacaseapi.domain.service.FeatureToggler;
+
+@Component
+public class AllocateTheCaseMidEventHandler implements PreSubmitCallbackHandler<AsylumCase> {
```

Comment:
Is there unit test coverage for this class -> AllocateTheCaseMidEventHandler

Reply:
Yes, here AllocateTheCaseMidEventHandlerTest.java and here RWA-318-Allocate-a-case-to-caseworker-midevent.json. Move onto this PR https://github.com/hmcts/ia-case-api/pull/742 to look for those files, please.